<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddBookDialog</class>
    <widget class="QDialog" name="AddBookDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Add New Book</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label_title">
                    <property name="text">
                        <string>Title:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="titleLineEdit"/>
            </item>
            <item>
                <widget class="QLabel" name="label_author">
                    <property name="text">
                        <string>Author:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="authorLineEdit"/>
            </item>
            <item>
                <widget class="QLabel" name="label_year">
                    <property name="text">
                        <string>Year:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QSpinBox" name="yearSpinBox">
                    <property name="minimum">
                        <number>1500</number>
                    </property>
                    <property name="maximum">
                        <number>2100</number>
                    </property>
                    <property name="value">
                        <number>2023</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="label_copies">
                    <property name="text">
                        <string>Number of Copies:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QSpinBox" name="copiesSpinBox">
                    <property name="minimum">
                        <number>1</number>
                    </property>
                    <property name="maximum">
                        <number>10000</number>
                    </property>
                    <property name="value">
                        <number>1</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>AddBookDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>AddBookDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
